src/tests/cpp/fwklib/Timer.hpp

```diff
@@ -28,18 +28,18 @@
 
 #ifdef _WIN32
 
-#include <time.h>
-#include <stdio.h>
-#include <stdlib.h>
+#include <ctime>
+#include <cstdio>
+#include <cstdlib>
 #include <windows.h>
 #include <winbase.h>
 
 #endif
 
 #ifdef _SOLARIS
 
-#include <time.h>
-#include <stdio.h>
+#include <ctime>
```

Comment:
Yeah.  Still kinda messy with all the platform `ifdef`s.  Because this code isn't related to public headers, which is the focus of GEODE-2439, I won't invest much time in fixing further.